plugins/plugins64.xml

```diff
@@ -1148,7 +1148,7 @@
 		<author>bycn82</author>
 		<homepage>https://github.com/bycn82/visimulator</homepage>
 		<versions>
-			<version number="17.12.17" md5="80bfd1abb48327992e26e0d4346784f6"/>
+			<version number="1.17" md5="80bfd1abb48327992e26e0d4346784f6"/>
```

Comment:
@oleg-shilo Seems you also modified a unrelated plugin. Could you please correct that.